css/index.css

```diff
@@ -172,4 +172,181 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+/* Media Query for Tablet devices */
+
+@media (max-width: 768px) {
+
+    .container {
+        width: 100%;
+    }
+
+
+    header nav {
+        width: 59.588%;
+        margin: 2% 0.5% 0.5% 0.5%;
+    }
+
+    header nav li {
+        flex: 1 1 auto;
+    }
+
+    header .logo {
+    margin: 1% 0 0 15%;
+}
+
+    .cta {
+    margin-top: 8%;
+    justify-content: center;
+}
+
+.cta .cta-text {
+    font-size: 4rem;
```

Comment:
Great usage of rem! This is something that we will be covering tomorrow, keep it up!